From 2f7f5a0e9c58016975c0f2a64e9b4098cc77fbf0 Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Mon, 8 Sep 2014 13:06:00 -0400
Subject: [tools] perf: Remove unused simple_strtoul() function

Message-id: <1410181752-25631-116-git-send-email-jolsa@redhat.com>
Patchwork-id: 91773
O-Subject: [PATCH RHEL7.1 BZ1134356 115/307] perf tools: Remove unused simple_strtoul() function
Bugzilla: 1134356
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Don Zickus <dzickus@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1134356
https://bugzilla.redhat.com/show_bug.cgi?id=1134356

upstream
========
commit a51e87cb5a0fbebee15a3373d951dbf6f59a76c2
Author: Ramkumar Ramachandra <artagnon@gmail.com>
Date: Tue Mar 18 17:05:15 2014 -0400

description
===========
Moreover, the corresponding function in include/linux/kernel.h is marked
obsolete.
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/perf/util/include/linux/kernel.h b/tools/perf/util/include/linux/kernel.h
index d8c927c..9844c31 100644
--- a/tools/perf/util/include/linux/kernel.h
+++ b/tools/perf/util/include/linux/kernel.h
@@ -94,12 +94,6 @@ static inline int scnprintf(char * buf, size_t size, const char * fmt, ...)
  return (i >= ssize) ? (ssize - 1) : i;
 }
 
-static inline unsigned long
-simple_strtoul(const char *nptr, char **endptr, int base)
-{
- return strtoul(nptr, endptr, base);
-}
-
 int eprintf(int level,
      const char *fmt, ...) __attribute__((format(printf, 2, 3)));
 
-- 
1.7.1